miri/lib.rs

```diff
@@ -157,7 +158,7 @@ pub fn eval_main<'a, 'tcx: 'a>(
         Ok(())
     }
 
-    let mut ecx = EvalContext::new(tcx, ty::ParamEnv::empty(traits::Reveal::All), limits, Default::default(), Default::default());
+    let mut ecx = EvalContext::new(tcx, ty::ParamEnv::empty(traits::Reveal::All), limits, Default::default(), Default::default(), Substs::empty());
```

Comment:
This does not compile in practice, but it's not my commit.

Reply:
cc @oli-obk - that's your commit

Reply:
Is this a `rustc` master vs nightly difference?

Reply:
It's https://github.com/solson/miri/tree/upstreamed2 which hasn't been merged in this repo yet. The corresponding PR was merged yesterday